app/assets/javascripts/Components/submission_file_manager.jsx

```diff
@@ -105,17 +134,32 @@ class SubmissionFileManager extends React.Component {
     });
   };
 
+  uploadFiles = (uploadTarget) => {
```

Comment:
I think this name is a bit misleading. You could do something like `openUploadModal`.